tools/functional-tester/etcd-tester/tester.go

```diff
@@ -110,7 +110,8 @@ func (tt *tester) runLoop() {
 				}
 			}
 			if !ok {
-				log.Printf("etcd-tester: [round#%d case#%d] checking current revisions failure...", i, j)
+				time.Sleep(time.Second)
```

Comment:
It names the directory by time stamp. There was conflict in the directory name `2016/02/06 23:29:00` with the same second.